test/helpers/vscodefake/extensioncontext.ts

```diff
@@ -38,8 +38,11 @@ export class DefaultExtensionContext implements vscode.ExtensionContext {
     get extensionMode(): vscode.ExtensionMode {
         throw new Error(notImplementedErr);
     }
+    extension: vscode.Extension<any>;
```

Comment:
I don't understand what the changes in this file are for or how they're used.

Reply:
Split to https://github.com/microsoft/vscode-cmake-tools/pull/2262